HISTORY.rst

```diff
@@ -4,12 +4,13 @@ History
 
 0.14.x
 ------
+* Creation of `time_bnds` variables when resampling data encoded with `CFTimeIndex` (non-standard calendars).
```

Comment:
At the moment, the functionalities are not used by the Indicator class. So I would change that to *Added utility functions to compute `time_bnds`*.

Reply:
Want me to reopen this PR and add the change? Alternatively, we could just hotfix that line on master. What do you think?

Reply:
No problem with hotfix.